### PR TITLE
rancher cluster SER-107

### PR DESCRIPTION
**Pull request by jgmartin**
docker-compose for a 2-service rancher cluster with load balancers

https://vitals.atlassian.net/browse/SER-107

### CODE REVIEW

**Comment by jgmartin** [comment]
@tobowers

File: cluster/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - rabbit1
+  environment:
+   - CLUSTERED=true
+   - CLUSTER_WITH=rabbit1
+   - RAM_NODE=true
+  ports:
+    - 5672
+    - 15672
+
+lb1:
```
Comment (tobowers) [inline cluster/docker-compose.yml:25]: let's name these a little better so we can use the name to update ui... maybe the rabbits themselves are rabbit-#{rabbit_version} and the loadbalancer is 'rabbit-1' ?

Comment (davelnewton) [inline cluster/docker-compose.yml:25]: Or `rabbit-master` etc.

Comment (tobowers) [inline cluster/docker-compose.yml:25]: yay @davelnewton :). I think we're going to move this bit into a private repo though :)

Comment (davelnewton) [inline cluster/docker-compose.yml:25]: I MAKE ONE LITTLE COMMENT!!1!

I AM THE DEATH OF OPEN SOURCE!

Comment (tobowers) [inline cluster/docker-compose.yml:25]: ha... nah - we will probably end up OSS that at some point... right now a lot of churn and it doesn't really apply to this repo as it's using a different image.

**Pull request closed without merging**